spec/controllers/admin/conferences_controller_spec.rb

```diff
@@ -165,6 +165,18 @@
       end
     end
 
+    describe 'GET #conference_wide_screen' do
+      it 'assigns the requested conference to conference' do
```

Comment:
If you use the cancancan ``load_and_authorize`` before filter, I'd say you can drop this test